docs/src/notebooks/demo.ipynb

```diff
@@ -45,6 +45,7 @@
    "source": [
     "import json\n",
     "import httpx\n",
+    "import requests\n",
```

Comment:
thanks for flagging this @emileten, I think the easier fix will be to replace `requests` with `httpx` in line 55 !